packages/components/addon/components/hds/dropdown/toggle-button.hbs

```diff
@@ -16,7 +16,7 @@
   @text={{this.text}}
   @icon="chevron-{{if @isOpen 'up' 'down'}}"
   @iconPosition="trailing"
-  @color={{@color}}
+  @color={{this.color}}
```

Comment:
@MelSumner I forgot to add also this in the previous review (will not throw otherwise)